server/controllers/finance/reports/income_expense/report.handlebars

```diff
@@ -75,8 +82,34 @@
           {{else}}
             {{>emptyTable columns=6}}
           {{/each}}
+          <tr>
+            <td colspan="2" class="text-right"><b>{{translate 'FORM.LABELS.TOTAL'}}</b></td>
+            <td class="text-right">{{debcred totals.expense.previousBalance ../metadata.enterprise.currency_id}}</td>
+            <td class="text-right">{{debcred totals.expense.balance ../metadata.enterprise.currency_id}}</td>
+            <td class="text-right">{{debcred totals.expense.difference ../metadata.enterprise.currency_id}}</td>
+            <td></td>
+          </tr>
         </tbody>
       </table>
+
+      <br/>
+      <table style="width:400px;background-color:#ddd;" class="pull-right table table-condensed table-report">
+        <tr>
+          <th colspan="2" class="text-center"><b>{{translate 'FORM.LABELS.PROFIT'}}</b></th>
+        </tr>
+        <tr>
+          <td>{{translate 'FORM.LABELS.EXPENSE'}}</td>
+          <td class="text-right">{{debcred totals.expense.balance ../metadata.enterprise.currency_id}}</td>
+        </tr>
+        <tr>
+          <td>{{translate 'FORM.LABELS.INCOME'}}</td>
+          <td class="text-right">{{debcred totals.income.balance ../metadata.enterprise.currency_id}}</td>
+        </tr>
+        <tr>
+          <td>{{translate 'FORM.LABELS.BALANCE'}}</td>
+          <td class="text-right">{{debcred totals.profits ../metadata.enterprise.currency_id}}</td>
```

Comment:
I think all these `../metadata.enterprise.currency_id` should just be `metadata.enterprise.currency_id`.  Have you tried this report with multiple enterprise currencies?  Do they change?